features/signup/src/main/java/com/kape/signup/utils/SignupScreenState.kt

```diff
@@ -30,6 +30,7 @@ val SUBSCRIPTIONS_FAILED_TO_LOAD = SignupScreenState(
     SignupStep.Subscriptions(supportsSubscription = true, displaySubscribeButton = false),
 )
 val NO_IN_APP_SUBSCRIPTIONS = SignupScreenState(loading = false, SignupStep.Subscriptions(false))
+val AMAZON_LOGIN = SignupScreenState(loading = false, SignupStep.Subscriptions(supportsSubscription = true, displaySubscribeButton = false))
```

Comment:
Note: supportsSubscription = false would lead to two empty (without price) product details for a monthly and a yearly being shown; we don't want that